cz.cvut.edu.comparch.qtrvsim.json

```
@@ -0,0 +1,31 @@
+{
+    "app-id" : "cz.cvut.qtrvsim",
+    "runtime" : "org.kde.Platform",
+    "runtime-version" : "6.2",
+    "sdk" : "org.kde.Sdk",
+    "command" : "qtrvsim_gui",
```

```suggestion
    "command" : "qtrvsim_gui",
    "rename-desktop-file": "qtrvsim.desktop",
    "rename-icon": "qtrvsim_gui",
```

(assuming these are the rename after decyphering the CMakeList.txt)